docroot/themes/contrib/civictheme/includes/cards.inc

```diff
@@ -28,7 +28,7 @@ function _civictheme_preprocess_node__card(&$variables) {
 function _civictheme_preprocess_node__civictheme_event_card(&$variables) {
   _civictheme_preprocess_node__card($variables);
   _civictheme_preprocess_node__date($variables);
-  _civictheme_preprocess_node__thumbnail($variables);
+  _civictheme_preprocess_node__thumbnail($variables, 'civictheme_promo_card');
```

Comment:
Added image_style argument to `_civictheme_preprocess_node__thumbnail` we should now be using the existing civictheme image styles for card components in drupal.